src/libraries/Native/native-binplace.proj

```diff
@@ -11,16 +11,18 @@
   <!-- Ordering matters! Overriding GetBinPlaceItems and Build targets after the Sdk import. -->
   <Target Name="GetBinPlaceItems">
     <ItemGroup>
-      <BinPlaceItem Include="$(NativeBinDir)*.dll" />
-      <BinPlaceItem Include="$(NativeBinDir)*.pdb" />
-      <BinPlaceItem Include="$(NativeBinDir)*.lib" />
-      <BinPlaceItem Include="$(NativeBinDir)*.a" />
-      <BinPlaceItem Include="$(NativeBinDir)*.bc" />
-      <BinPlaceItem Include="$(NativeBinDir)*.so" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dbg" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dylib" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dwarf" />
-      <FileWrites Include="@(BinPlaceItem)" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dll" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.pdb" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.lib" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.a" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.bc" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.so" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dbg" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dylib" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dwarf" />
+
+      <BinPlaceItem Condition="'$(BinPlaceTestRuntimePack)' != 'true'" Include="@(NativeBinPlaceItems)" />
+      <FileWrites Include="@(NativeBinPlaceItems)" />
```

Comment:
Not understanding why we need a `FileWrites` here. This target doesn't write files at this location, it's reading files from this location.

Reply:
<FileWrites> was there before our changes, so we thought it mattered.

Reply:
Adding them to the items above they will be written into the bin directory using the bin place logic. We might not need the `FileWrites` here, but this is how it was setup already. @ericstj do you know?

Reply:
As written though it unconditionally reports the writes even though we conditionally do the bin place. That seems wrong.

Reply:
We don't Conditionally do the bin place. Either `BinPlaceItem` or `NativeBinPlaceItems` will be bin placed. When `NativeBinPlaceItems` are going to be bin placed, `BinPlaceItem` will not and the other way around.